### ISSUE TITLE
Can the model from Huggingface be used for the quality estimation?

### ISSUE THREAD

**Issue by p4vv37** [microsoft/CodeBERT#242]
After loading the model with code:
`    config = T5Config.from_pretrained("microsoft/codereviewer")
    model = ReviewerModel.from_pretrained("microsoft/codereviewer", config=config)`
I'm getting the error:
`Some weights of ReviewerModel were not initialized from the model checkpoint at microsoft/codereviewer and are newly initialized: ['cls_head.weight', 'cls_head.bias']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.`

Does it mean, that the model from Huggingface cannot be used for the quality estimation?

**Comment by celbree** [comment]
No. The pre-trained model needs fine-tuning for the quality estimation task.

**Comment by Qinhaifu** [comment]
I had the same problem, I download chekpoint from HuggingFace.
"The pre-trained checkpoint of CodeReivewer is available in [Huggingface](https://huggingface.co/microsoft/codereviewer)."
I use these check point directly , and then get the same error.

What are the requirements for fine-tuning with the official data set ? 3090 is OK?
"Before you start to run experiments with CodeReviewer, please download the [datasets](https://zenodo.org/record/6900648) first."?

**Comment by Qinhaifu** [comment]
After I fine-tuning, how to use the finetuned model and checkpoint that from HuggingFace together?

**Comment by fasterinnerlooper** [comment]
I would assume that you would have to load the weights separately for each task. Merging models is a task that has been well-researched so you should be able to find a lot of documentation online for this.
